docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md

```diff
@@ -0,0 +1,69 @@
+
+### OpenCypher-compatible query language
+
+The native Nebula Graph Query Language, also known as nGQL, is a declarative, openCypher-compatible textual query language. It is easy to understand and easy to use. For more information, see [nGQL guide](https://github.com/vesoft-inc/nebula-docs/tree/master/docs_2.0/manual_EN/4.ngql-guide).
+
+### Easy data modeling and high flexibility
+
+You can easily model the connected data into Nebula Graph for your business without forcing them into a structure such as a relational table, and properties can be added, updated, and deleted freely. For more information, see [Data modeling](TODO) [TODO].
+
+### Reliable access control
+
+Nebula Graph supports strict role-based ACL and external authentication servers such as LDAP (Lightweight Directory Access Protocol) servers to enhance data security. For more information, see [Authentication and authorization](TODO) [TODO].
+
+### High scalability
+
+Nebula Graph is designed in a shared-nothing architecture and supports scaling in and out without interrupting the database service.
+
+### High popularity
+
+Nebula Graph is being used by tech leaders such as Tencent, Vivo, Meituan, JD Digits. For more information, visit the [Nebula Graph official website](https://nebula-graph.io/).
+
+## Use cases
+
+Nebula Graph can be used to support various graph-based scenarios. To spare the time spent on pushing the kinds of data mentioned in this section into relational databases and on bothering for join queries, use Nebula Graph.
+
+### Fraud detection
+
+Financial institutions have to traverse countless transactions to piece together potential crimes and understand how combinations of transactions and devices might be related to a single fraud scheme. This kind of scenario can be modeled in graphs, and with the help of Nebula Graph, fraud rings and other sophisticated scams can be easily detected.
+
+### Real-time recommendation
+
+Nebula Graph offers the ability to instantly process the real-time information produced by a visitor and make accurate recommendations on articles, videos, products, and services.
+
+### Intelligent question-answer system
+
+Natural languages can be transformed into knowledge graphs and stored in Nebula Graph. A question organized in a natural language can be resolved by a semantic parser in an intelligent question-answer system and re-organized. Then, possible answers to the question can be retrieved out of the knowledge graph and provided to the one who asked the question.
```

Comment:
can be retrieved out of ==> is "out of" necessary or correct?

Reply:
Used "from" instead. Thx!